### ISSUE TITLE
copyright and license

### ISSUE THREAD

**Issue by jGaboardi** [pysal/momepy#305]
* license
  * `martinfleis/momepy` was licensed as MIT where PySAL stuff is BSD-3 Clause
* copyright
  *  PySAL stuff is (generally) stamped with a PySAL Developers copyright

These should likely be update in `momepy`.

**Comment by martinfleis** [comment]
MIT and BSD-3 Clause are compatible, right? So we can change it and release 0.6 under BSD-3 Clause with no issues. I am always a bit lost in these things.

We will need to make a release under BSD before the next meta to ensure there's no conflict.

Regarding the copyright, I am personally fine with that but again, not sure if we can simply replace the existing one.

**Comment by jGaboardi** [comment]
Yes, I am pretty sure the license can simply be swapped out prior to the next release, which should be before the next meta release, as you say. I also think the copyright can be updated like they have in [coin-or/pulp](https://github.com/coin-or/pulp/blob/master/LICENSE) for something like:

> Copyright (c) 2018-2021 Martin Fleischmann
> Modifications Copyright (c) 2022- PySAL Developers

Let's get other opinions on this -- @sjsrey @ljwolf @knaaptime @slumnitz @pysal/devs 

Also, perhaps @ocefpaf will have some insight on this?

**Comment by ocefpaf** [comment]
> Also, perhaps @ocefpaf will have some insight on this?

Nothing to add. We only need to pay attention to update the metadata in the conda package later when the new release is out.

**Comment by jGaboardi** [comment]
> > Also, perhaps @ocefpaf will have some insight on this?
> 
> 
> 
> Nothing to add. We only need to pay attention to update the metadata in the conda package later when the new release is out.

Thanks for the input!

**Comment by ljwolf** [comment]
Copyright change is fine with me. I'm also fine with @martinfleis retaining copyright!

**Comment by jGaboardi** [comment]
Alright, Martin. It's all your call now.
